### ISSUE TITLE
"No Active LSP" after :LspInstall

### ISSUE THREAD

**Issue by alechp** [ChristianChiarulli/LunarVim#387]
### Quick fix

"No LSP Install" seems to have been resolved after:
1. `:PackerSync`
2. `:LspInstall rust`
> Have only tested this on rust so far

### Error Logs

1. After installing rust 
```
:LspInstall rust
:LspInfo
```

![Screen Shot 2021-05-10 at 7 01 05 PM](https://user-images.githubusercontent.com/3058939/117747949-f6114c80-b1c3-11eb-8353-450d5edf5d34.png)

2. Check the LunarVim settings to make sure it's not commented out

![Screen Shot 2021-05-10 at 7 14 22 PM](https://user-images.githubusercontent.com/3058939/117748014-1e994680-b1c4-11eb-9666-11164138096a.png)

3. Check to see if there are any weird pathing issues
```
:lua print(vim.fn.stdpath('cache'))
```

![Screen Shot 2021-05-10 at 7 05 10 PM](https://user-images.githubusercontent.com/3058939/117748047-28bb4500-b1c4-11eb-8592-c20f190965d7.png)

4. Finally, ran Sync and re-installed
```
:PackerSync
:LspInstall rust
:LspInfo
``` 

![Screen Shot 2021-05-10 at 7 10 08 PM](https://user-images.githubusercontent.com/3058939/117748187-6ae48680-b1c4-11eb-96bb-0524d5d01c74.png)

### Why I made this thread
If the problem's resolved, why post this?

1. Figured others might benefit since I found a couple threads like this one:
- https://github.com/ChristianChiarulli/LunarVim/issues/300
- https://github.com/ChristianChiarulli/LunarVim/issues/292

2. There does seem to be some fundamental issue regardless because:
- I definitely installed these language servers more than once
- I have definitely ran PackerSync more than once

The only thing I'm not sure of is whether or not I previously ran PackerSync and LspInstall in quick succession. Whatever the case, something definitely seems odd.

--- 

TLDR: 
This is obviously my mistake, not LunarVim's, but thought it useful to document/share regardless.
